_posts/2024-12-19-cosmicqc.md

```diff
@@ -0,0 +1,285 @@
+Once images were grouped together, they performed a method using KDE to sample the images into their respective phenotypes.
+They then trained an SVM (Support Vector Machine) per phenotype group, to classify where there are artifacts and where there are cells.
+I believe I barely scratch the surface of the complexities behind this method, so feel free to read the paper if you want a more in-depth look.
+
+What I really liked about this paper is that it is attempting to preserve images that include artifact, as these images still contain other cells.
+It is different than the current process where whole images are removed that fail QC metrics like blur and over-saturation (traditional done in CellProfiler).
+But, I also have my suspicions about using cells that come from an image with corruption, such as an artifact.
+I wonder how much things like an artifact or smudge could be directly impacting the biology the cells.
+In my opinion, I would rather default to losing those cells in an image than risk any issues in downstream interpretation of the results do to any impacts the artifacts in an image had.
+
+I have two main concerns from this paper, specifically for it's generalizability and usability in my own pipelines:
+
+1. It was not clear to me how this method is filtering out artifacts at the single-cell level. From what I understood from the figures, the SVM are able to detect where there is artifacts and where there are cells, but I didn't see where in their pipeline to they remove segmentations from the CellProfiler output. This is a big concern for generalizability as it seems like you would need to train SVMs on your data to be able to apply this method but there isn't a direct link to that output and filtering cells.
+2. The code is available via FigShare, but it was a bit hard to find and parse. Since it is not on GitHub, that also limits the usability since there isn't any environment or dependencies I could find and there isn't a way to create issues and contact the maintainers for assistance. As well, I found that the code I could find was not easy to understand and lacked documentation. I want to make clear, I appreciate that they made the code available and open source which is more than some other papers. My main point is that I don't think this method is feasible to include in my standard workflow.
+
+In conclusion, I want to thank the authors of this paper for their hard work and their important contributions to pushing the field towards developing quality control methodology at the single-cell level for morphology profiles.
+This paper fueled the fire that I had started in myself about wanting to pursue working on a new methodology, one that is simple, to filter out poor quality segmentations from single-cell morphology profiles.
```

Comment:
Do you know if there are quality control methods for bulk morphology profiles? This might be an important paragraph to add. If not, you can discuss anomaly detection, which is the opposite side of the same coin as quality control. I know of at least Assaf and Erin's preprint - they might have some citations in there for us to explore.

Reply:
I don't see anything in particular, but I am now finding that Juan's paper from back in 2017 mentions cell-level quality control, so I think I want to focus on adding a paragraph for that. 

I don't think adding anomaly detection will help the blog, maybe only add more confusion since anomaly and outlier can be considered the same thing.